### DISCUSSION TITLE
Custom Search Engines

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferdium/ferdium-app#1704] [Ideas]
Web browsers such as Chrome allow you to easily add custom search engines. Please add the setting to add a custom search engine inside of Ferdium instead of being limited to the search engine options inside the app.

When adding a search engine, you will be asked for the following information...

- Name
- URL (With %s) added in it which will be replaced with the search query

**Comment by vraravam** [comment]
@trymeouteh  - please do not raise feature requests via the discussion board. We will not be able to support such requests. you can raise feature requests by following the template that's been setup in the Issues tracker
